### PR TITLE
Update plan-for-multi-geo.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Microsoft 365 Multi-Geo eDiscovery configuration link to page for more information

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4327

File: microsoft-365/enterprise/plan-for-multi-geo.md [hunk 1]
```diff
@@ -52,7 +52,9 @@ You can configure any of the following locations to be satellite locations where
 
 [!INCLUDE [Microsoft 365 Multi-Geo locations](../includes/microsoft-365-multi-geo-locations.md)]
 
-When you configure multi-geo, consider taking the opportunity to consolidate your on-premises infrastructure while migrating to Microsoft 365. For example, if you have on-premises farms in Singapore and Malaysia, then you can consolidate them to the APC satellite location, provided data residency requirements allow you to do so.
+When you configure multi-geo, consider taking the opportunity to consolidate your on-premises infrastructure while migrating to Microsoft 365. For example, if you have on-premises farms in Singapore and Malaysia, then you can consolidate them to the APC satellite location, provided data residency requirements allow you to do so. 
+
+For more information about edisovery configuration, We recommend that you review [Microsoft 365 Multi-Geo eDiscovery configuration](https://docs.microsoft.com/microsoft-365/enterprise/multi-geo-ediscovery-configuration)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/plan-for-multi-geo.md]:
```suggestion
For more information about eDiscovery configuration, please review [Microsoft 365 Multi-Geo eDiscovery configuration](https://docs.microsoft.com/microsoft-365/enterprise/multi-geo-ediscovery-configuration).
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**